### ISSUE TITLE
Allow full GDrive links in spreadsheet import

### ISSUE THREAD

**Issue by jpolitz** [brownplt/code.pyret.org#392]
Right now, you have to figure out exactly which bit of the URL is the ID to use for importing spreadsheets.

We should have a `load-spreadsheet` that takes a full GDrive URL and parses it to get the ID.

**Comment by schanzer** [comment]
@jpolitz @ds26gte I've made this a more general request, since there are other places where we want a fileID or folderID -- and ideally none of them should 100% rely on a user carefully selecting only the ID when they copy/paste

**Comment by jpolitz** [comment]
Just wanted to comment that I tried this out today and, at least for spreadsheets, the full URL is currently accepted, and in fact the Bootstrap starter code already does it!

(CC @shriram since we talked about this today)

![image](https://github.com/brownplt/code.pyret.org/assets/119891/42c9edbf-8e0b-4536-a994-702be2978a4e)

I'm happy leaving the issue open for other kinds of links (specifically for contexts/import for `gdrive`), but this is not an issue with spreadsheets

**Comment by jpolitz** [comment]
(You can even kind of tell the regex they use; if you edit the URL prefix it fails but it can have any suffix :-))

![image](https://github.com/brownplt/code.pyret.org/assets/119891/b64c3114-5152-41bf-81d6-57c7b44d2178)

**Comment by shriram** [comment]
Wow, okay, LOL. So the old version will continue to work, but the full URL
also works? This is best.

Can you please update the docs?

**Comment by jpolitz** [comment]
Yup, and plain ID works too, checked and updated (https://github.com/brownplt/pyret-docs/commit/6d85789a995abf8f6375cc7516406987315972d1)

**Comment by schanzer** [comment]
Oooh! Does this mean I can rip out the code that handles this in the Data Science context? Or is there a merge that needs to happen on CPO first?

**Comment by jpolitz** [comment]
Oh let me double-check before you do that. I thought that `load-spreadsheet` was _the_ `load-spreadsheet`.
